src/main/java/com/zaxxer/nuprocess/windows/WindowsProcess.java

```diff
@@ -45,8 +45,10 @@
 {
    private static final boolean IS_SOFTEXIT_DETECTION;
 
-   private static final int BUFFER_SIZE = 65536;
+   // See https://github.com/JetBrains/jdk8u_jdk/blob/master/src/windows/native/java/lang/ProcessImpl_md.c#L36-L41
+   private static final int BUFFER_SIZE = 4096 + 24;
```

Comment:
If you'd prefer, I could also change this to be something like:
```java
private static final int BUFFER_SIZE = Integer.parseInt(System.getProperty("com.zaxxer.nuprocess.pipebuffer", "65536"));
```

That would preserve the existing large buffer, but give me the option of configuring a smaller value in Bitbucket Server using a system property. In practice, though, I suspect very few applications will notice the difference in this change to the default. I've run testing that forks thousands of processes and the test duration is identical overall before and after reducing the buffer size.